recipes/iqtree/build.sh

```diff
@@ -1,11 +1,19 @@
 #!/bin/bash
-export CFLAGS="-I$PREFIX/include"
-export LDFLAGS="-L$PREFIX/lib"
+export CFLAGS="$CFLAGS -I$PREFIX/include"
+export LDFLAGS="$LDFLAGS -L$PREFIX/lib"
 export CPATH=${PREFIX}/include
 
+if [ "$(uname)" == Darwin ]; then
+    export MACOSX_DEPLOYMENT_TARGET=10.7
+    export CMAKE_TRY_COMPILE_TARGET_TYPE=STATIC_LIBRARY
```

Comment:
Hmm, the latest circleci run implies that this change didn't get picked up - it appears to still be building a test executable...

Reply:
You know, it might be the case that this flag can only be set with the `-D CMAKE_TRY_COMPILE_TARGET_TYPE=STATIC_LIBRARY` syntax, rather than as an env var.

Reply:
Let's see what happens in the next build.  The `patch` directive was not indented properly underneath `source` 🤦‍♂

Reply:
Looks like something is still off with the patch, can you double-check that?

Reply:
Do you still want to run any tests re: `CMAKE_TRY_COMPILE_TARGET_TYPE=STATIC_LIBRARY` ?

Reply:
Not just yet, let's focus on resolving one thing at a time right now - let's finish our 10.9 checks.

Reply:
Okay, now that 10.9 is sorted out, can you check and see if this flag is necessary, too?

Reply:
PS - this "parametric" development cycle is painful - sorry about that!

Reply:
To clarify, you just want to test with `export  CMAKE_TRY_COMPILE_TARGET_TYPE=STATIC_LIBRARY`? commented out?

Reply:
All checks have passed without this flag.